src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_general.html.twig

```diff
@@ -87,6 +87,13 @@
             {{ form_widget(generalForm.default_status) }}
           </div>
         </div>
+        <div class="form-group row">
+          {{ ps.label_with_help(('Default specific price priorities'|trans), ('Specific prices are applied based on this priority list to all products unless product has its own specific list configured.'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
```suggestion
          {{ ps.label_with_help(('Default order of priority for specific prices'|trans), ('If a customer meets multiple conditions, specific prices will be applied in this order of priority, unless a different order has been set for a particular product.'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

I think this message should be a help message instead of a tooltip, as it gives users essential instructions for completing a task.

Reply:
Agreed, I think the way it's used here displays it as a help under the input This is "old style" forms

Reply:
so this also goes to "out of scope" pr to adapt old style configuration forms i guess?